Steer/DigitizerWorkflow/src/FT0DigitizerSpec.cxx

```diff
@@ -119,20 +120,19 @@ class FT0DPLDigitizerTask
         LOG(INFO) << "For collision " << collID << " eventID " << part.entryID << " source ID " << part.sourceID << " found " << hits.size() << " hits ";
 
         // call actual digitization procedure
-        labels.clear();
         mDigitizer.setEventID(collID);
         mDigitizer.setSrcID(part.sourceID);
         mDigitizer.process(&hits);
-        // copy labels into accumulator
-        labelAccum.mergeAtBack(labels);
       }
       mDigitizer.setDigits(mDigitsBC, mDigitsCh);
+      // copy labels into accumulator
+      //   labelAccum.mergeAtBack(labels);
```

Comment:
this looks weird. So does it mean that the digitizer itself accumulates labels for all events? There is no flushing in between? Probably the same is done for digits?